samples/cpp/chat_sample/chat_sample.cpp

```diff
@@ -15,7 +15,12 @@ int main(int argc, char* argv[]) try {
     
     ov::genai::GenerationConfig config;
     config.max_new_tokens = 10000;
```

Comment:
```suggestion
    config.max_new_tokens = 100;
```
Please, do the same for python
By the forth question TinyLlama-1.1B-Chat-v1.0 collapses to repeat just a single word